automatic/sonarlint-vs2019/sonarlint-vs2019.nuspec

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>sonarlint-vs2019</id>
+    <title>SonarLint for Visual Studio 2019</title>
+    <version>4.16.0.14364</version>
+    <authors>SonarSource</authors>
+    <owners>BBT Software AG</owners>
+    <projectSourceUrl>https://github.com/SonarSource-VisualStudio/sonarlint-visualstudio</projectSourceUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/bbtsoftware/chocolatey-packages/tree/master/icons/sonarlint-vs2019.png</iconUrl>
+    <projectUrl>https://marketplace.visualstudio.com/items?itemName=SonarSource.SonarLintforVisualStudio2019</projectUrl>
+    <packageSourceUrl>https://github.com/bbtsoftware/chocolatey-packages/tree/master/automatic/sonarlint-vs2019</packageSourceUrl>
+    <licenseUrl>https://github.com/SonarSource-VisualStudio/sonarlint-visualstudio/blob/master/LICENSE</licenseUrl>
```

Comment:
Should be https://github.com/SonarSource/sonarlint-visualstudio/blob/master/LICENSE